### ISSUE TITLE
Unable to delete stream with deployed on dataflow with skipper

### ISSUE THREAD

**Issue by cppwfs** [spring-cloud/spring-cloud-dataflow#2443]
Spring cloud data flow 1.6.2.RELEASE local
Skipper 1.0.9.RELEASE
Created the following streams:
```
stream create f --definition ":foo > transform --expression=payload+'-sample1' | log" --deploy
stream create b --definition ":bar > transform --expression=payload+'-sample2' | log" --deploy
stream create r --definition "http --server.port=9000 | router --expression=payload.contains('a')?':foo':bar'" --deploy
```
Could not delete the 'r' stream.
I had to flush my db so that I could get back into a demo-able mode.    Can't replicate.

**Comment by ilayaperumalg** [comment]
@cppwfs I have tried this a few times and couldn't replicate this issue so far. When you get to see this again, please check if you see any stack trace etc., from either SCDF or Skipper log.

**Comment by markpollack** [comment]
Another bit of info was that the Darwin GA app starters for rabbit were used in the original report.  The router sink appears to be broken.  Will create another issue.  That said, with the `r` stream partially deployed, I was able to delete it. Switching to use Celsius SR3, I discovered a typo in the `r` stream, needed another quote.

```
stream create r --definition "http --server.port=9000 | router --expression=payload.contains('a')?':foo':'bar'" --deploy
```

That deployed fine and I was able to delete it.  Closing the issue and opening one for the router in Darwin failing to start.
